app/stac_api/serializers.py

```diff
@@ -439,6 +461,22 @@ def update(self, instance, validated_data):
         )
         return super().update(instance, validated_data)
 
+    def update_or_create(self, validated_data, **kwargs):
+        """
+        Create and return a new `Collection` instance, given the validated data.
```

Comment:
Docstring needs to be updatd. Something like "If the collection already exists, it will be updated, otherwise a new instance will be created" or the like.

Reply:
good point